polyfill/spatial-navigation-polyfill.js

```diff
@@ -961,6 +961,29 @@
     }
   }
 
+  /**
+   * Get distance between the search origin and a candidate element along the direction when candidate element is inside the search origin.
+   * @see {@link https://wicg.github.io/spatial-navigation/#select-the-best-candidate}
+   * @function getDistanceFromPoint
+   * @param point {Point} - The search origin
+   * @param element {DOMRect} - A candidate element
```

Comment:
is it DOM? or DOMRect?

Reply:
The type of parameter element is DOMRect (https://developer.mozilla.org/en-US/docs/Web/API/DOMRect).